src/frontend/next/src/components/SearchHelp/SearchHelp.tsx

```diff
@@ -0,0 +1,74 @@
+import { withStyles, makeStyles } from '@material-ui/core/styles';
+import InfoOutlinedIcon from '@material-ui/icons/InfoOutlined';
+
+import { Typography, Tooltip } from '@material-ui/core';
+
+const useStyles = makeStyles(() => ({
+  infoButton: {
+    marginTop: '7.5px',
+    marginLeft: '-25px',
```

Comment:
I'm getting the button appearing off screen with this:

![2021-01-20_21-25-21](https://user-images.githubusercontent.com/7242003/105271785-21fdb900-5b66-11eb-91b4-616aeb4bc55d.gif)

Reply:
Just removed it! However I'm not sure if this marginLeft exist with a reason? Since it's working with other search components. Let me look into it.

Reply:
Yeah it's weird. It might be for when the actual icon gets imported to wherever it belongs. We'll probably have to come back to this at a later date.